mission_control/src/mission_control.cpp

```diff
@@ -203,66 +132,53 @@ void MissionControlNode::reportHeartbeat(const ros::TimerEvent& timer)
   pub_report_heartbeat.publish(outmsg);
 }
 
+void MissionControlNode::executeMissionT(const ros::TimerEvent& timer)
+{
+  NodeStatus missionStatus = m_mission_map[m_current_mission_id]->getMissionStatus();
+  if (missionStatus == NodeStatus::IDLE || missionStatus == NodeStatus::RUNNING)
+  {
+    missionStatus = m_mission_map[m_current_mission_id]->executeMissionTickEvent();
```

Comment:
@patapufate if the mission fails, the default abort behavior should also kick in.

Reply:
fixed.